src/ontology/fbbt-edit.obo

```diff
@@ -87559,15 +87559,17 @@ creation_date: 2019-02-12T09:51:54Z
 [Term]
 id: FBbt:00048152
 name: bitter-sensing neuron of the labellum
-def: "Gustatory receptor neuron of the labellum that responds to bitter stimuli and innervates a bitter-sensitive sensillum. All of these cells express Gr66a, usually in combination with Gr32a, Gr33a and Gr89a (Weiss et al., 2011)." [FlyBase:FBrf0212834]
+def: "Gustatory receptor neuron of the labellum that responds to bitter stimuli and innervates a bitter-sensitive sensillum. All of these cells express Gr66a, usually in combination with Gr32a, Gr33a and Gr89a (Weiss et al., 2011). They also express the inotropic receptor Ir25a and respond to high concentrations of salts (Liman et al., 2014; Freeman and Dahanukar, 2015; Lee et al., 2018)." [FlyBase:FBrf0212834, FlyBase:FBrf0229299, FlyBase:FBrf0229608, FlyBase:FBrf0237619]
```

Comment:
I agree the way I phrased it is misleading as it seems to imply Ir25a is what confers the sensitivity to high salt, which is not what I meant (and is not what the authors meant as well). I'll remove the bit about Ir25a expression.

I do think the assertion that these neurons *also* respond to high salt (without being the only high salt-sensing neurons) is supported:

* Bitter-sensing neurons in the labellum are found in S- and I-type sensilla (Freeman and Dahanukar, 2015);
* "L2" neurons in S- and I-type sensilla are both high salt-sensitive and bitter-sensitive (Liman et al., 2014; Jaeger et al., 2018 – which I should have cited here).

But I'll admit this may be interpreting too much, so I'll update the definition to state that *some* of those neurons also respond to high salts.